recipes/afwizard/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "afwizard" %}
+{% set version = "1.0.0b4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/afwizard-{{ version }}.tar.gz
+  sha256: 7754f55c2ffd5ad6b27123d803db765491f3212de735b38fb6b6ebb437bb4f9b
+
+build:
+  entry_points:
+    - afwizard = afwizard.__main__:main
+    - extract_opals_schema = afwizard.opals:_automated_opals_schema
+    - upgrade_filter_library = afwizard.library:upgrade_filter_library
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+  run:
+    - afwizard-library
+    - click
+    - gdal
```

Comment:
This is fixed upstream to 3.4.0. Which one is correct, since you're also upstream?

Reply:
I have recently fixed the underlying problem that introduced the requirement for the pin. The version used in the recipe does not require it anymore.